### DISCUSSION TITLE
Split dashboards, the other tabs and addons

### DISCUSSION THREAD

**Discussion by Xitee1** [home-assistant/frontend#14467] [Accessibility]
My list with dashboards and addons keeps growing and slowly it becomes more and more a mess and I have to search more often for the tab which I want.
My suggestion is that dashboards, the other tabs (Logbook, History, Map,... - hope that was correct - I have it on german) and Ingress addons get split and have headings to keep the side panel more organized.

I know that I can move the entries around but I have no possibility to add headings and group them. They also only get stored in the browser.

For example somewhat like this:

**_Dashboards_**
Overview
Cameras
OtherDashboard1
OtherDashboard2
Energy

**_Tools?_**
Map
Logbook
History
File Editor
Terminal
Media

**_Addons/Ingress?_**
HACS
Grafana
motionEye
InfluxDB

**Comment by matthiasdebaat** [comment]
This could be useful when you have a lot of menu items. For example when you have 3 or more of the same category show a title.
